### DISCUSSION TITLE
Switch

### DISCUSSION THREAD

**Discussion by Mephisto090** [Zefau/ioBroker.jarvis#1922] [Q&A]
Hej,

Ich habe das jetzt mal per Hand umgesetzt um es zu veranschaulichen. Zwei Fragen. 
Gibt es eine Möglichkeit das im Status Deaktiviert ein Button ist, im Step 2 nichts zur Auswahl steht und im Step 3 eine Textbox ist?
Zur anderen Frage. Wie bekomme ich die leere Textbox größer? (Siehe Step 3)

Step 1:
![image](https://user-images.githubusercontent.com/85995176/192844143-909e5fa2-8814-4d5a-9b26-0bbbb3c93a12.png)

Step 2:
![image](https://user-images.githubusercontent.com/85995176/192845140-f630b4cc-155b-41e1-8178-eef31ce90972.png)

Step 3:
![image](https://user-images.githubusercontent.com/85995176/192845323-ace60292-ad6c-4f16-b896-33aa9c371a3e.png)

Passwort zum deaktivieren ist ein anderes Objekt als die Aktivierung der Alarmanlage

![image](https://user-images.githubusercontent.com/85995176/192847983-a66dc20f-0157-44f3-804a-86643eea2d90.png)

Viele Grüße

**Comment by mcuiobroker** [comment]
>Gibt es eine Möglichkeit das `im Status` Deaktiviert ein Button ist

Nein. Aber man kann in Abhängigkeit vom Status, einen Button anzeigen lassen (wie oben zu sehen ist) oder halt nicht.

Wie, mit dem display Attribut.

**Reply by Mephisto090** [reply]
Wäre das nicht ein Verbesserungsvorschlag? Ich meine ich könnte sowas sicherlich öfter gebrauchen. Warum nicht andere auch?

@Zefau

**Reply by Mephisto090** [reply]
> Wie, mit dem display Attribut.

https://github.com/Zefau/ioBroker.jarvis/issues/1923
Der Zusammenhang für andere :)

**Comment by mcuiobroker** [comment]
InputAction vergrößern geht nicht, es war glaube ich mal angedacht die Anzahl der Zeichen (inputAction) in den Optionen vorzugeben, bislang nicht vorhanden.
![image](https://user-images.githubusercontent.com/84818934/192980684-add41741-701a-4a5b-a5b0-8526e4014302.png)
